tests/system/API/ResponseTraitTest.php

```diff
@@ -29,8 +29,16 @@
  */
 final class ResponseTraitTest extends CIUnitTestCase
 {
+    /**
+     * @var mixed|MockIncomingRequest|null
```

Comment:
Why was `mixed` added?

```suggestion
     * @var MockIncomingRequest|null
```

Reply:
Because it not typed, so it will be mixed by default, then it nullable, so it nullable, I am going to revert it as union mixed should be forbidden change.

Reply:
I got it. But the rule is not good.
The code base has a lot of not typed properties, but it must not be `mixed` in most cases.

Reply:
Yes, I created PR https://github.com/rectorphp/rector-src/pull/3160 to patch it in rector-src side.